packages/react-native-reanimated/Common/cpp/reanimated/LayoutAnimations/LayoutAnimationsUtils.h

```diff
@@ -19,18 +19,23 @@ struct Rect {
 
 struct Frame {
   std::optional<double> x, y, width, height;
+  int count = 0;
   Frame(jsi::Runtime &runtime, const jsi::Object &newStyle) {
     if (newStyle.hasProperty(runtime, "originX")) {
       x = newStyle.getProperty(runtime, "originX").asNumber();
+      count++;
     }
     if (newStyle.hasProperty(runtime, "originY")) {
```

Comment:
We should add early returns or use `else if` instead. Once this is done, we can just a boolean to store the information whether there is some custom style or not.
```suggestion
    else if (newStyle.hasProperty(runtime, "originY")) {
```

Reply:
We can't, the purpose of this class is to aggregate non-props updates in layout animations. We need to parse all of them, and we need to count them all, so that we can calculate how many props updates are there in a given layout animation.